src/CommandDialog.h

```diff
@@ -0,0 +1,53 @@
+#ifndef CONTROLDIALOG_OPENMSX_H
+#define CONTROLDIALOG_OPENMSX_H
+
+#include "ui_CommandDialog.h"
+#include <QDialog>
+
+
+struct CommandRef
+{
+    QString name;
+    QString description;
+    QString source;
+    QString icon;
+    int index;
+
+    bool operator<(const CommandRef& ref) const {
+        return index < ref.index;
+    }
```

Comment:
Maybe I'm wrong, but I don't see this `operator<` being used anywhere in the code. Could this be a leftover from storing these in a map?